lib/models/container.js

```diff
@@ -5,11 +5,16 @@ var EventEmitter = require('events').EventEmitter
 var mapKeys = require('object-loops/map-keys')
 var NotModifiedError = require('./base-store').NotModifiedError
 var Promise = require('bluebird')
+var stream = require('stream')
 var util = require('util')
 var utils = require('../utils')
 
+var sampleStats = require('./stats-sample.json')
+
 module.exports = Container
 
+var streams = new Map()
```

Comment:
`Map` doesn't exist in node `0.10`, so you won't be able to use this... Facebook has the (immutable)[https://github.com/facebook/immutable-js/] library that includes a Map - that's the only one I know off hand though...